### ISSUE TITLE
Fetch the eosio account from an API

### ISSUE THREAD

**Issue by ghost** [Gimly-Blockchain/eosio-did-resolver#2]
Complete the function
```js
async function fetchAccount(did: string, parsed: ParsedDID, options: DIDResolutionOptions): Promise<object> {
    // Find the API from registered eosio chains or from options

    // Fetch the eosio account
    return {};
}
```

https://github.com/Gimly-Blockchain/eosio-did-resolver/blob/master/src/resolver.ts#L7

**Comment by Balurat** [comment]
Because I fetch the chain data from the eosio-did-chain-registry within the checkDID method, I added an additional parameter to this function. That way we don't have to parse the method-specific-id twice. 
The new parameter contains an object with two fields:

`function fetchAccount(registryEntry: object, did: string, parsed: ParsedDID, options: DIDResolutionOptions)`
```
registryEntry = {
   chain: object    // contains the chain data from the registry
   subject: string  // is the account name
}
```
For details see the pull request #7

**Comment by ghost** [comment]
Resolved in #10
